src/Application/Cli/TravisAutoDeployCommand.php

```diff
@@ -58,13 +58,6 @@ protected function configure()
                 getcwd()
             )
             ->addOption(
-                'php-version',
-                null,
-                InputOption::VALUE_REQUIRED,
-                'Specify for which php version you want to deploy documentation, mainly to avoid multiple deploys',
-                '5.4'
-            )
```

Comment:
Yes but users of Couscous might be using it, so even though we will not use it ourselves we must not break this